test/commons/dom/get-composed-parent.js

```diff
@@ -0,0 +1,72 @@
+/* global xit */
+describe('dom.getComposedParent', function () {
+  'use strict';
+  var getComposedParent = axe.commons.dom.getComposedParent;
+  var fixture = document.getElementById('fixture');
+  var shadowSupport = document.body && typeof document.body.attachShadow === 'function';
```

Comment:
This line is only shadow DOM v1, so it should only be used when we are sure that we don't need tests for shadow DOM v0